### PR TITLE
Earn Coins Button

### LINKED ISSUE #204: Earn Coin Color Change
**Is your feature request related to a problem? Please describe.**
The earn coins button is not the desired colors

**Describe the solution you'd like**
Change the background color to the secondary color and the font to the base color.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Changed colors
2. Added PokeDollar SVG

## Purpose
Looks better

## Approach
SCSS

Closes #204

### CODE REVIEW

**Review by Gjhnsn — APPROVED** [review]
Keepin that color scheme consistent! Looks good!

**Review by Lap343 — APPROVED** [review]
Nice code. Dont see any issues

File: pokeapp/src/components/Navigation.js [hunk 1]
```diff
@@ -49,10 +58,20 @@ const Navigation = ({ isMyPoke, isPokeDex, setIsMyPoke, setIsPokeDex, currencyDi
...
           <div className="coin-link-total">
-            <h4 className="coin-link-font">${pokeDollars}</h4>
+            <h4 className="coin-link-font">
+              <img
+                src={pokeDollar}
+                alt="pokeDollar"
+                className="poke-dollar-icon"
+              />
+              {pokeDollars}
+            </h4>
```
Comment (BlackCubes) [inline pokeapp/src/components/Navigation.js:68]: Interesting

**Review by BlackCubes — APPROVED** [review]
Noice job!!!

**Review by Lap343 — APPROVED** [review]
Mostly code styling. Good work

**Pull request merged**